arrows/ffmpeg/ffmpeg_audio_stream_settings.h

```diff
@@ -0,0 +1,49 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+/// \file
+/// Declaration of FFmpeg audio stream settings.
+
+#ifndef KWIVER_ARROWS_FFMPEG_FFMPEG_AUDIO_STREAM_SETTINGS_H_
+#define KWIVER_ARROWS_FFMPEG_FFMPEG_AUDIO_STREAM_SETTINGS_H_
+
+#include <arrows/ffmpeg/ffmpeg_util.h>
+#include <arrows/ffmpeg/kwiver_algo_ffmpeg_export.h>
+
+extern "C" {
+#include <libavcodec/avcodec.h>
+}
+
+namespace kwiver {
+
+namespace arrows {
+
+namespace ffmpeg {
+
+// ----------------------------------------------------------------------------
+struct KWIVER_ALGO_FFMPEG_EXPORT ffmpeg_audio_stream_settings
+{
+  ffmpeg_audio_stream_settings();
+  ffmpeg_audio_stream_settings( ffmpeg_audio_stream_settings const& other );
+  ffmpeg_audio_stream_settings( ffmpeg_audio_stream_settings&& other );
+
+  ~ffmpeg_audio_stream_settings();
+
+  ffmpeg_audio_stream_settings&
+  operator=( ffmpeg_audio_stream_settings const& other );
+  ffmpeg_audio_stream_settings&
+  operator=( ffmpeg_audio_stream_settings&& other );
+
+  int index;
```

Comment:
Is there a reason for public data members?

Reply:
This `struct` is used only to pass these data items from input to output, with perhaps occasional use-case-specific editing going on in between. We could, of course, add setters and getters for each member and make them private, but in the absence of any complex behavior being implemented by this `struct` that would necessitate a layer of abstraction, I opted not to -- seemed unnecessary at the time. You may see I made similar decisions in e.g. `struct ffmpeg_video_settings`.

Reply:
Since those structures are defined in the header files, a comment explaining public data members, e.g. "used only to pass these data items from input to output, with perhaps occasional use-case-specific editing going on in between", would be nice.